src/main/java/com/suse/saltstack/netapi/datatypes/Job.java

```diff
@@ -1,17 +1,68 @@
 package com.suse.saltstack.netapi.datatypes;
 
+import com.google.gson.annotations.SerializedName;
 import java.util.List;
 
+// todo - handle dates too (they are represented in really
```

Comment:
yeees, but I'm too lazy to use shift :D out of curiosity:which ide do you use? (intellij idea works fine with it)